src/main/java/seedu/duke/Parser.java

```diff
@@ -0,0 +1,161 @@
+package seedu.duke;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Parser {
```

Comment:
I read through already, I think its ok no problem wait for the rest then maybe can merge as a base first. 💯